sample-app/e2e/functional.test.ts

```diff
@@ -0,0 +1,174 @@
+    await fillPaymentDetailsSheet()
+    await element(by.id(Selectors.TOKEN_CVV_CODE)).typeText(
+      TestData.SECURITY_CODE,
+    )
+    await element(by.id(Selectors.TOKEN_SCROLL_VIEW)).scrollTo('bottom')
+    await element(by.id(Selectors.PAY_WITH_TOKEN)).tap()
+    await complete3DS2()
+    try {
+      await element(by.text(Selectors.THREEDS2_COMPLETE_BUTTON)).longPress()
+      await element(by.text(Selectors.THREEDS2_COMPLETE_BUTTON)).longPress()
+    } catch (error) {}
+    await assertResultsScreen('1', '1')
+  })
+
+  it('should successfully complete a 3DS2 token pre-auth transaction', async () => {
+    await element(by.id(Selectors.FEATURE_LIST)).scrollTo('bottom')
+    await element(by.text(Selectors.TOKEN_PAYMENTS)).tap()
+    await element(by.text(Selectors.TOKENIZE_NEW_CARD)).tap()
+    await fillPaymentDetailsSheet()
+    await element(by.id(Selectors.TOKEN_CVV_CODE)).typeText(
+      TestData.SECURITY_CODE,
+    )
+    await element(by.id(Selectors.TOKEN_SCROLL_VIEW)).scrollTo('bottom')
+    await element(by.id(Selectors.PREAUTH_WITH_TOKEN)).tap()
+    await complete3DS2()
+    try {
+      await element(by.text(Selectors.THREEDS2_COMPLETE_BUTTON)).longPress()
+      await element(by.text(Selectors.THREEDS2_COMPLETE_BUTTON)).longPress()
+    } catch (error) {}
+    await assertResultsScreen('2', '1')
+  })
+
+  it('should successfully complete a 3DS2 payment transaction', async () => {
+    await element(by.text(Selectors.PAY_WITH_CARD)).tap()
+    await fillPaymentDetailsSheet()
+    await complete3DS2()
+    await assertResultsScreen('1', '1')
+  })
+
+  it('should successfully complete a 3DS2 pre-auth transaction', async () => {
+    await element(by.text(Selectors.PAY_WITH_PREAUTH)).tap()
+    await fillPaymentDetailsSheet()
+    await complete3DS2()
+    await assertResultsScreen('2', '1')
+  })
+
+  it('should successfully complete a 3DS2 register card transaction', async () => {
+    await element(by.text(Selectors.REGISTER_CARD)).tap()
+    await fillPaymentDetailsSheet()
+    await complete3DS2()
+    await assertResultsScreen('3', '1')
+  })
+
+  it('should successfully complete a 3DS2 check card transaction', async () => {
+    await element(by.text(Selectors.CHECK_CARD)).tap()
+    await fillPaymentDetailsSheet()
+    await complete3DS2()
+    await assertResultsScreen('4', '1')
+  })
+
+  // it('should successfully complete a 3DS2 payment transaction via payment methods', async () => {
+  //   await element(by.text(Selectors.PAYMENT_METHODS)).tap()
+  //   await addCardPaymentMethod()
+  //   // Will fail until CSC toggle is implemented
+  //   await assertResultsScreen('1', '1')
+  // })
+
+  // it('should successfully complete a 3DS2 pre-auth transaction via payment methods', async () => {
+  //   await element(by.text(Selectors.PREAUTH_METHODS)).tap()
+  //   await addCardPaymentMethod()
+  //   // Will fail until CSC toggle is implemented
+  //   await assertResultsScreen('1', '1')
+  // })
```

Comment:
`Will fail until CSC toggle is implemented` - it is implemented, 
![IMG_6302](https://github.com/Judopay/JudoKit-ReactNative/assets/85407970/d487f4c9-f642-4810-9ae1-3fa27a7e9369)

Reply:
I think we have a bug then, as I don't see the CSC sheet to enter the CV2 come up when trying a payment via payment methods, can you try on your end?